### DISCUSSION TITLE
pc端无法登录

### DISCUSSION THREAD

**Discussion by zhouyingocean** [ZuodaoTech/everyone-can-use-english#916] [Q&A]
每次点github登录都显示网络错误

**Comment by an-lee** [comment]
参考这个： [总是报网络错误怎么办？](https://1000h.org/enjoy-app/faq.html#%E6%80%BB%E6%98%AF%E6%8A%A5%E7%BD%91%E7%BB%9C%E9%94%99%E8%AF%AF%E6%80%8E%E4%B9%88%E5%8A%9E)

**Comment by bigfish4u** [comment]
升级到v0.3.0-0.3.3这几个版本都无法通过github账号登录，显示network error；只能回退到0.2.3使用。

**Reply by an-lee** [reply]
> 升级到v0.3.0-0.3.3这几个版本都无法通过github账号登录，显示network error；只能回退到0.2.3使用。

0.2.3 能连上？这跟版本应该没关系啊，就是域名被 block 了。
